IM150/admin/admin_forum_tour.php

```diff
@@ -163,7 +165,7 @@ function reorder_pages()
 			$message = prepare_message(trim($message), 1, 1, 1, $bbcode_uid);
 
 			$sql = "UPDATE " . FORUM_TOUR_TABLE . "
-				SET page_subject = '$subject', page_text = '$message', page_access = $page_access
+				SET page_subject = '$subject', page_text = '$message', page_access = '$page_access'

```

Comment:
The information being updated in the database would be a string. It could hold numbers or characters in the Subject and body of the post.

So my best assumption would be string since it is not only holding numbers, it needs to be able to hold letters and special char as well.